components/EntryList.js

```diff
@@ -0,0 +1,55 @@
+import Divider from "./Divider";
+import { Fragment } from "react";
+import Tab from "./Tab";
+import Tabs from "./Tabs";
+/* import Badge from "./Badge"; */
```

Comment:
Did you accidentally comment this out?
You are using an element called `badge` (in lowercase) in line 26 and 33. You probably want to use the actual `Badge` component for which you would need the import.

Reply:
Yes, I wanted to test something with this Badge and forgot to uncomment it. I did it now.